src/components/Onboarding/kit/FileUploaderField/index.js

```diff
@@ -111,11 +126,7 @@ export const FileUploaderField = ({
             margin-top: ${1 * GU}px;
           `}
         >
-          {description || (
-            <div>
-              Drag and drop a file here or <TextFileUploader />.
-            </div>
-          )}
+          {description}
```

Comment:
I am afraid of the changes we are doing to this component as it was used to upload the Agreement document already. And working as expected.

Reply:
First of  all if you take a look to the desings are not matching with the agreement ones 
![Screen Shot 2021-09-01 at 10 15 43 PM](https://user-images.githubusercontent.com/11763623/131766056-885291ff-828d-4bc4-9017-99f74b02b1d7.png)

<https://www.figma.com/file/ySHlXMwz5elSXKI3feisJU/Gardens-wizard-creation-UI?node-id=495%3A6144>

and in order to be a reusable component we should not be hardcoding those kind of stuff the description should be passed as a parameter as is being done here <https://github.com/1Hive/honey-pot/blob/2848a3f3ab11a76d6a573db5606f8a03eca9e997/src/components/Onboarding/Screens/Apps/AgreementSettings/index.js#L155>

anyways we can take a look afterwards